spec/models/user_spec.rb

```diff
@@ -65,13 +28,7 @@
 
   describe '#role?' do
     it 'checks if a role is included' do
-      Role.create({ name: "Admin" })
-      user = User.create!({
-                             name: 'Alex',
-                             password: 'testpassword',
-                             email: 'fake@fake.com',
-                             role_ids: [1]
-                           })
+      user = User.create!(attributes_for(:user, name: 'Adam', role_ids: [1]))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.